### ISSUE TITLE
Option for complete Instance configuration share

### ISSUE THREAD

**Issue by Chris87-ux** [ioBroker/ioBroker.modbus#258]
Hi @GermanBluefox,
great work with this addapter, i have 9 instances running with it now and there are mostly working perfect.

Never the less, I get several requests to share my "modbus-lists" which works fine if only Holding or input is used, but if you using all 4 chambers it is quiet dificult, as well don't miss the configuration ( Aliasses activated, device ID, summ to 16Bit,....).

 So, to sum up it would be nice , to share the complete config of a instance.
Thanks again
/Chris

**Comment by Apollon77** [comment]
You mean "download it" or what?

**Comment by Chris87-ux** [comment]
> You mean "download it" or what?

yes, in some kind of file, that others can upload it and have the same configuration as I had.
